tgui/packages/tgui/styles/components/Section.scss

```diff
@@ -24,25 +24,20 @@ $separator-color: colors.$primary !default;
   }
 }
 
-.Section__title {
-  position: relative;
+.Section__head {
+  display: flex;
+  flex-flow: row nowrap;
   padding: 0.5em;
   border-bottom: base.em(2px) solid $separator-color;
```

Comment:
why?

Reply:
i have no clue why this thing is how it is

Reply:
im pretty sure this was some sort of section header dropdown fix